Nos/Views/NoteComposer/NoteComposer.swift

```diff
@@ -185,6 +221,43 @@ struct NoteComposer: View {
         .alert(unwrapping: $alert)
     }
 
+    /// Action Bar displayed below the editing controller.
+    private var composerActionBar: some View {
+        ComposerActionBar(
+            editingController: $editingController,
+            expirationTime: $expirationTime,
+            isUploadingImage: $isUploadingImage,
+            showPreview: $showNotePreview
+        )
+    }
+
+    /// Note Preview displayed above the editing controller when the Preview switch is turned on.
+    private func buildNotePreview(for note: Event) -> some View {
```

Comment:
I understand the thinking behind using the word "build" here, but I'd like to avoid it and make this more concise. We have several `func`s throughout the codebase now that return `some View` and none use "build" in the name, so removing it here would be more consistent with what we have.
```suggestion
    private func notePreview(for note: Event) -> some View {
```